package.json

```diff
@@ -24,31 +24,37 @@
   "main": "./app",
   "scripts": {
     "start": "DEBUG=raneto node ./example/server.js",
-    "postinstall": "npm update && ./node_modules/.bin/bower install"
+    "postinstall": "npm update && node_modules/gulp/bin/gulp.js"
```

Comment:
Is there are a reason to omit the `./`? (like Windows compatibility?)
Prefer to keep it there as it indicates (to me, maybe others), that gulp.js is a standalone executable.